### PR TITLE
Bump content-api-models to 10.22

### PR DESCRIPTION
**Pull request by tomrf1**
https://github.com/guardian/content-api-models/pull/88

### CODE REVIEW

**Comment by shtukas** [comment]
:+1:

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## 10.22
+* Bump content-api-models (refactored circe macros)
```
Comment (LATaylor-guardian) [inline CHANGELOG.md]: This shouldn't say `bump content-api-models` as the consumer doesn't know what that means.

Comment (LATaylor-guardian) [inline CHANGELOG.md]: `Refactored circe macros to ... (explain concisely)` should be fine.

**Comment by LATaylor-guardian** [comment]
👍

**Pull request merged**